learning/tour-of-beam/backend/internal/storage/adapter.go

```diff
@@ -0,0 +1,146 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"fmt"
+
+	tob "beam.apache.org/learning/tour-of-beam/backend/internal"
+	"cloud.google.com/go/datastore"
+)
+
+func sdk2Key(sdk tob.Sdk) string {
```

Comment:
Mmm.... maybe sdkToKey or just toKey will be better
I just don't like use "2" instead of to :)